PLRPC/ApiRepositoryImpl.cs

```diff
@@ -1,21 +1,23 @@
 ﻿using System.Text.Json;
 using LBPUnion.PLRPC.Types.Entities;
+using LBPUnion.PLRPC.Types.Enums;
+using LBPUnion.PLRPC.Types.Interfaces;
 
-namespace LBPUnion.PLRPC.Types;
+namespace LBPUnion.PLRPC;
 
 public class ApiRepositoryImpl : IApiRepository
 {
-    private readonly int cacheExpirationTimeMs;
+    private readonly int cacheExpirationTime;
```

Comment:
Since you’re removing the ms maybe you should make it a TimeSpan and then use the milliseconds of the timespan in the cache expiration